README.md

```diff
@@ -16,6 +16,12 @@ You can install yourdfpy directly from pip:
 pip install yourdfpy
 ```
 
+## Visualization
+
+Once installed, your can visualize a URDF model from the command line:
```

Comment:
Just a tiny `your` -> `you` typo :-)

Reply:
Arh, classic :sweat_smile: Thank you.

```suggestion
Once installed, you can visualize a URDF model from the command line:
```

(Feel free to go ahead and suggest/apply/merge, I absolutely don't mind :smiley:)